### DISCUSSION TITLE
Doesn't work on RHEL 8

### DISCUSSION THREAD

**Discussion by Cgratier** [datastax/diagnostic-collection#171] [Q&A]
Hello, 
As the code is using ntp et ntpstat , both of them are not compatible with  RHEL 8. 
Do you have a solution for it ? 

Reagrds,
Cédric

**Comment by michaelsembwever** [comment]
ntpstat and ntpq are used, but are optional commands.
https://github.com/datastax/diagnostic-collection/blob/master/ds-collector/rust-commands/collect-info.rs#L783C8-L804 

do you know what the RHEL8 equivalent for those^ lines are ?
